src/Controller/Controller.php

```diff
@@ -191,14 +191,14 @@ public function __construct(
     ) {
         if ($name !== null) {
             $this->name = $name;
-        } elseif (!$this->name) {
+        } elseif (!isset($this->name)) {
```

Comment:
Static analyzers hate doing isset checks on non-nullable properties. Is this the route we want to go for typed properties?

Reply:
I had checked with phpstan's highest level and this change does not cause any error. Using `isset()` to check initialization of non nullable properties is the right thing to do IMO.

psalm does whine about this and report it as redundant check. I had opened an issue for it but they said it's a design choice which won't be changed. I'll just disable the related rule for psalm once it's running again.

Reply:
Isn't using isset() going to result in different behavior than before? Previously this block would run if `name = ''` which is the current default. Now this block will not run and $name won't get a fallback value based on the `controller` parameter.

Reply:
I have also removed the default empty string for the property.

Reply:
Oops I missed that :cry: